assembly-multiuser/assembly-ide-war/pom.xml

```diff
@@ -230,7 +230,7 @@
                                 <include>IDE.jsp</include>
                             </includes>
                         </overlay>
-                        <overlay></overlay>
+                        <overlay />
```

Comment:
<!-- empty groupId/artifactId represents the current build -->
see more https://maven.apache.org/plugins/maven-war-plugin/overlays.html